src/tests/SearchResults.test.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import SearchResults from "../components/SearchResults";
+
+describe("SearchResults", () => {
+  const validProps = {
+    results: ["i1", "i2, i3"],
+  };
+
+  it("renders correctly", () => {
+    const { asFragment } = render(
+      <SearchResults results={validProps.results} />
+    );
+
+    expect(asFragment()).toMatchSnapshot();
+  });
```

Comment:
Another test could be added here to check the number of images is the same as the length of the array passed in:
```
it("renders the correct number of images", async () => {
    render(<SearchResults results={validProps.results} />);
    const images = await screen.getAllByAltText("mock");

    expect(images).toHaveLength(validProps.results.length);
  });
```